.github/workflows/continuous-integration-workflow.yml

```diff
@@ -51,16 +51,12 @@ jobs:
           push: true
           tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ secrets.DOCKERHUB_REPOSITORY }}:latest
   job-three:
-    name: Deploy Heroku
+    name: Run Azure Login with OpenID Connect
```

Comment:
This name doesn't feel quite right; this job is just hitting an endpoint secured with credentials directly in the URL